bot.js

```diff
@@ -741,8 +741,25 @@ function command(message) {
         );
         log.info("Help is for command: " + target);
         if (Object.keys(config.helpTopics).includes(target)) {
-            const detailsObject = config.helpTopics[target];
+            let detailsObject = config.helpTopics[target];
             let response;
+            if (detailsObject.alias != null) {
+                // This command is an alias, try to find the help text object for the actual command
+                if (config.helpTopics.hasOwnProperty(detailsObject.alias)) {
+                    detailsObject = config.helpTopics[detailsObject.alias];
+                } else {
+                    log.notice(
+                        "Command " +
```

Comment:
[Nitpick] This should probably say "help topic" - One is technically allowed to define non-command topics.